tests/api.c

```diff
@@ -14840,6 +14841,56 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
 #endif
 } /* END test_wc_PKCS7_EncodeEncryptedData() */
 
+/*----------------------------------------------------------------------------*
+ | hash.h Tests
+ *----------------------------------------------------------------------------*/
+static void test_wc_HashInit(void) 
+{
+    /*enum for holding supported algorithms, #ifndef's restrict if disabled*/
+    enum wc_HashType enumArray[] = {
+    #ifndef NO_MD5
+            WC_HASH_TYPE_MD5,
+    #endif
+    #ifndef NO_SHA
+            WC_HASH_TYPE_SHA,
+    #endif
+    #ifndef WOLFSSL_SHA224
+            WC_HASH_TYPE_SHA224,
+    #endif 
+    #ifndef NO_SHA256  
+            WC_HASH_TYPE_SHA256,      
+    #endif 
+    #ifndef WOLFSSL_SHA384  
+            WC_HASH_TYPE_SHA384,      
+    #endif     
+    #ifndef WOLFSSL_SHA512  
+            WC_HASH_TYPE_SHA512,      
+    #endif     
+    };
+    int ret = 0; /*0 indicates tests passed, 1 indicates failure*/
+    int enumlen = (sizeof(enumArray))/4;/*dynamically finds the length*/
```

Comment:
"sizeof(enum wc_HashType)" should be used here instead of "4".

Also, please add some whitespace after the ";" and before your comment, thanks.

Reply:
thank you